### ISSUE TITLE
Matrix.to needs a favicon

### ISSUE THREAD

**Issue by JorikSchellekens** [matrix-org/matrix.to#131]
Currently no favicon is provided

**Comment by 532910** [comment]
And not only 16px, but also high resolution for HiDPI screens.

**Comment by ShadowJonathan** [comment]
I'd recommend this to be the default matrix logo, it would make the most sense

**Comment by niquewoodhouse** [comment]
How about using the exact same favicons as matrix.org? Eg
![favicon-32x32](https://user-images.githubusercontent.com/1587166/140085150-9bcfbe61-f514-4e05-bf9f-9627605616aa.png)

**Comment by 532910** [comment]
Nope, it's ugly, wo HiDPI, dark theme support and should be updated:

![image](https://user-images.githubusercontent.com/14617699/140085936-22bdddaa-be85-4e9a-a404-2b6f8af75061.png)

**Comment by niquewoodhouse** [comment]
![image](https://user-images.githubusercontent.com/1587166/140393069-e2a62579-2a4d-4741-bffc-f368475447c6.png)

[Figma link](https://www.figma.com/file/Po6qWY7kC3xpvwv29hFtpX/Matrix.org-favicon?node-id=2%3A17)

**Comment by ShadowJonathan** [comment]
*any* favicon would be an improvement over the current situation, adding the same favicon as matrix.org should suffice at the moment, and high quality versions can be added afterwards
